src/qt/pivx/masternodewizarddialog.cpp

```diff
@@ -50,10 +59,20 @@ MasterNodeWizardDialog::MasterNodeWizardDialog(WalletModel *model, QWidget *pare
     setCssProperty(ui->labelMessage1a, "text-main-grey");
     setCssProperty(ui->labelMessage1b, "text-main-purple");
 
+    QString collateralAmountStr = GUIUtil::formatBalance(clientModel->getMNCollateralRequiredAmount());
+    ui->labelMessage1a->setText(tr("To create a PIVX Masternode you must dedicate %1 (the unit of PIVX) to the network (however, these coins are still yours and will never leave your possession).")
+                                   .arg(collateralAmountStr) + "<br>" + "<br>" +
+                                   tr("You can deactivate the node and unlock the coins at any time."));
+
```

Comment:
why no formatting here?
```suggestion
    ui->labelMessage1a->setText(formatHtmlContent(
                formatParagraph(tr("To create a PIVX Masternode you must dedicate %1 (the unit of PIVX) "
                        "to the network (however, these coins are still yours and will never leave your possession).").arg(collateralAmountStr)) +
                formatParagraph(tr("You can deactivate the node and unlock the coins at any time."))));
```

Reply:
No apparent reason, just added it.